src/03_policy_assignments/00_backend.tfvars

```
@@ -0,0 +1,2 @@
+bucket = "tfapporg-terraform-state"
+prefix = "eng-gcp-governance"
```

```suggestion
prefix = "eng-gcp-governance/policy_assignments"
```
tf state for assignment